python/ray/worker.py

```diff
@@ -1848,6 +1855,14 @@ def method(self):
         assert "CPU" not in resources, "Use the 'num_cpus' argument."
         assert "GPU" not in resources, "Use the 'num_gpus' argument."
 
+    accelerator_type = kwargs.get("accelerator_type")
+
+    if accelerator_type and num_gpus is None:
+        logger.warning(
+            f"Accelerator type {accelerator_type} was specified, but"
+            "`num_gpus` was not set. This will be scheduled on a node with the"
+            "accelerator type, but Ray will not reserve the accelerator.")
```

Comment:
Why not raise ValueError?

Reply:
actually upon thinking about it further, this warning probably shouldn't even exist since it's valid to specify `num_gpus` via `.options()`